python/ml4ir/applications/classification/tests/generate_test_data.py

```diff
@@ -0,0 +1,73 @@
+# Generating csv test data for classification model.
+
+from random import seed
+from random import randint
+
+import csv
+
+
+CSV_TRAIN_FILE_PATH = "data/csv/train/file_0.csv"
+CSV_TEST_FILE_PATH = "data/csv/test/file_0.csv"
+CSV_VALIDATION_FILE_PATH = "data/csv/validation/file_0.csv"
+COLUMNS_HEADER = ["query_key", "query_text", "domain_id", "user_context", "entity_id"]
+QUERY_VOCABULARY = [
+    "the", "tragedy", "of", "hamlet", "prince", "denmark", "shakespeare", "homepage", "entire", "play", "act", "i",
+    "scene", "elsinore", "a", "platform", "before", "castle", "francisco", "at", "his", "post", "enter", "to", "him",
+    "bernardo", "whos", "there", "nay", "answer", "me", "stand", "and", "unfold", "yourself", "long", "live", "king",
+    "he", "you", "come", "most", "carefully", "upon", "your", "hour", "tis", "now", "struck", "twelve", "get", "thee",
+    "bed", "for", "this", "relief", "much", "thanks", "bitter", "cold", "am", "sick", "heart", "have", "had", "quiet",
+    "guard", "not", "mouse", "stirring", "well", "good", "night", "if", "do", "meet", "horatio", "marcellus", "rivals",
+    "my", "watch", "bid", "them", "make", "haste", "think", "hear", "ho", "friends", "ground", "liegemen", "dane"
+]
+
+FEATURE_1_VOCABULARY = [str(i) for i in range(0, 20)]
```

Comment:
@Ullimague Can you rename accordingly here?

Also, might be useful to use the same vocab for both?